CompanionApp/Companion/Views/Onboarding/WalkthroughIllustrationPage.xaml

```diff
@@ -5,20 +5,20 @@
     
     <local:WalkthroughIllustrationStepTemplate
 		StepBackgroundImage="https://d3nz6a95rn0zd2.cloudfront.net/v3.0/walkthrough_illustration_01.png"
```

Comment:
Same here - if this cloudfront url isn't used anymore, delete it through this file.